thresholdkey_generator.go

```diff
@@ -19,7 +20,7 @@ import (
 //               with Applications to Electronic Voting
 //               Aarhus University, Dept. of Computer Science, BRICS
 type ThresholdKeyGenerator struct {
-	nbits                          int
```

Comment:
[L28](https://github.com/keep-network/paillier/blob/0d2a123af76417b75dd91eb1927575f5aa0fb6f2/thresholdkey_generator.go#L28) needs to be updated.